src/Compute/Compute/Generated/ProximityPlacementGroup/ProximityPlacementGroupGetMethod.cs

```diff
@@ -123,6 +124,11 @@ public override void ExecuteCmdlet()
         [SupportsWildcards]
         public string Name { get; set; }
 
+        [Parameter(
+            Position = 2,
+            ValueFromPipelineByPropertyName = true)]
```

Comment:
- why add a position attribute to a switch parameter?
- position attribute is valid for the parameter sets?